build/nested-cluster-base/dockerd-entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
This is [almost] a direct copy. I don't think there's really a simpler way to do this, unless docker itself has changed. Maybe @BenTheElder can speak to that.

Reply:
my "nicer" way is to just use the stock init script in debian (ok patched to not try to remount cgroups unnecessarily but otherwise), this could probably still use some comments though.